test/spec/singleStart.spec.js

```diff
@@ -16,7 +16,7 @@ describe('CodeEditor', function() {
   });
 
 
-  (singleStart ? it.only : it)('should render', async function() {
+  (singleStart ? it.only : it.skip)('should render', async function() {
```

Comment:
This skip needs to be removed, right?

Reply:
We already have a render test, I skipped this one as we do an evaluation through console.log and I didn't want it to affect the logs, but in singleStart it's nice to see it in the browser logs at least.